conda/recipes/ucxx/recipe.yaml

```
@@ -328,7 +326,10 @@ outputs:
     requirements:
       host:
         - pip
-        - python
+        # Pin this to a specific Python because we only need one build of it,
+        # then ignore the `python_abi` run export so that it can be used with
+        # any version of `ucxx`
+        - python =3.12
```

Can we build with the oldest instead? I have more faith in forward compatibility if it ever becomes relevant.
```suggestion
        - python =3.10
```